composeApp/src/jsMain/kotlin/io/github/karczews/brainagator/tts/TextToSpeech.js.kt

```
@@ -0,0 +1,116 @@
+    override fun stop() {
+        try {
+            getSpeechSynthesis().cancel()
+        } catch (e: Exception) {
+            console.error("TTS Stop Error: ${e.message}")
+        }
+    }
+
+    override fun isSpeaking(): Boolean =
+        try {
+            getSpeechSynthesis().speaking
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)